src/main/java/androidx/view/TextView.kt

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (C) 2018 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.view
+
+import android.text.Editable
+import android.text.TextWatcher
+import android.widget.EditText
+import android.widget.TextView
+
+/**
+ * Set and Get the string value of EditText content
+ */
+var TextView.content
+    get() = text.toString()
+    set(value) {
+        text = value
+    }
+
+/**
+ * @returns Whether the EditText text is empty or not
+ */
+val TextView.isEmpty get() = length() == 0
+
+/**
+ * @returns Whether the EditText text is blank or not
+ */
+val TextView.isBlank get() = text.isBlank()
+
+/**
+ * Clears the contents of the EditText
+ */
+fun TextView.clear() {
```

Comment:
I'm not sure these 4 extensions have any real value. They're conveniences for things which are already pretty convenient.